website/docs/components/page-header/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,237 @@
+
+<Hds::PageHeader as |PH|>
+  <PH.Title>HCP Packer Dashboard</PH.Title>
+  <PH.IconTile @icon="packer-color" @color="packer" />
+  <PH.Description>
+    Channel created and managed automatically for you by Packer. For more information on how this channel is managed refer to the Packer <Hds::Link::Inline @icon="external-link" @href="#">documentation</Hds::Link::Inline>.
+  </PH.Description>
+</Hds::PageHeader>
+
+!!!
+
+!!! Insight
+
+**Hand-off note:** Nesting a [Link Inline](/components/link/inline) with an icon inside of the description is not supported in the Figma component. Instead, override the description text with the correct text style and color, and ensure that this detail is accounted for in the engineering hand-off. 
+
+!!!
+
+## Custom metadata
+
+If necessary to include metadata like key/value pairs, multiple page-level statuses, or other structured content, custom metadata can be passed to the component via the `customMetadata` property in Figma.
+
+![Page Header custom metadata](/assets/components/page-header/page-header-custom-metadata-example.png =700x*)
+
+### Key/value pairs as custom metadata
+
+Representing metadata with a set of key/value pairs is common in HashiCorp products and can be useful when communicating relational information between products, versioning, and other structured data.
+
+![Custom metadata key/value pairs](/assets/components/page-header/page-header-custom-metadata.png =600x*)
+
+!!! Dont
+
+We recommend not exceeding more than four key/value pairs, this can result in unnecessarily complex content with the Page Header. Instead, move this content into the page.
+
+![Complex metadata dont](/assets/components/page-header/page-header-custom-metadata-dont.png)
+
+!!!
+
+## Actions
+
+Use [Buttons](/components/button) and [Dropdowns](/components/dropdown) in the Page Header to highlight functions and actions that impact the page as a whole. Examples of this include:
+
+- Creating new objects that are listed within the page
+- Surfacing management details like connecting to an API
+- Deleting or deactivating an object
+- Pausing or refreshing a service pertaining to the page
+
+![Page Header Actions](/assets/components/page-header/page-header-actions.png =853x*)
+
+### Primary button
+
+Use a `primary` [Button](/components/button) in the Page Header to highlight the most important flow or action on the page. Examples of this include:
+
+- A flow that creates an object that is related to the page, e.g., creating a new cluster or adding a new user.
+- Launching or deploying a specific product instance.
+
+### Secondary button
+
+Use a `secondary` [Button](/components/button) in the Page Header when highlighting actions that are less important than the primary action, such as editing the content on a page, managing access, etc.
+
+### Tertiary button
+
+Use a `tertiary` [Button](/components/button) when highlighting low-priority actions like linking to external documentation.
```

Comment:
Do we have an example in production that uses this pattern? 
Since the page header space is precious, only critical and page-level actions should be included in the actions area. I think the primary and secondary button option is enough, and there is no need to support a tertiary button.

A link to the documentation page can be included in the page description as an inline link (like the Packer example), or in the main page area in the empty state call-to-action area.  What do you think?

Reply:
Can this be updated to use a more real-world example? I know there was one that came up in Nomad. My concern is that this example will lead to confusion since you've outlined elsewhere that external documentation links should go in the description.

Reply:
I've updated the example of this, the example from Nomad triggers a terminal instance that allows you interact with the job through a console.

Alternatively, since this isn't really a pattern we want to recommend broadly, we could exclude this from the docs. The support would still exist within the component, but would be "hidden" a bit. Downsides: might be confusing when to use a tertiary button (if at all).